src/ucm/util/sys.c

```diff
@@ -340,3 +343,18 @@ char *ucm_concat_path(char *buffer, size_t max, const char *dir, const char *fil
 
     return buffer;
 }
+
+ucs_status_t ucm_get_mem_type_current_device_info(ucs_memory_type_t memtype, ucs_sys_bus_id_t *bus_id)
+{
+    ucs_status_t status = UCS_OK;
+    ucm_event_installer_t *event_installer;
+
+    ucs_list_for_each(event_installer, &ucm_event_installer_list, list) {
+        status = event_installer->get_mem_type_current_device_info(bus_id, memtype);
```

Comment:
isn't it segfault if try with ROCM installer is fist in the list? need to check if func ptr NULL